pyproject.toml

```diff
@@ -30,6 +29,8 @@ de-dwd = {version = "^1.0.1", optional = true}
 pypresseportal = {version= "^0.1", optional = true}
 de-nina = {version= "^1.0.1", optional = true}
 de-ladestationen = {version= "^1.0.4", optional = true}
+onnxruntime = "^1.10.0"
+numpy = "~1.21.0"
```

Comment:
NumPy is not compatible with NumPy `^1.22.0` because this project still supports Python 3.7. Once you decide to drop support for Python 3.7, you can also update NumPy.